app/assets/javascripts/historical.js.es6

```diff
@@ -0,0 +1,153 @@
+class HistoricalChart {
+  constructor(data, chartElement) {
+    this.data = data;
+    this.chartElement = chartElement;
+    this.dimensions = {};
+    this.filters = {};
+    this.counts = [];
```

Comment:
I don't think this needs to be an instance variable on `this` - it's only used locally in the `render()` function.